### PR TITLE
LPD-29337 Resolve test failures in DatabaseUpgradeClient

### PR DESCRIPTION
**Pull request by javalosjr96**

### CODE REVIEW

**Comment by javalosjr96** [comment]
ci:test:upgrade

**Comment by achaparro** [comment]
@javalosjr96 please send it to database-infra repo to be tested

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase [hunk 1]
```diff
@@ -119,7 +119,7 @@ definition {
 
 		var schemaVersion = ValidateSmokeUpgrade.getSchemaVersion(mysqlStatement = "SELECT schemaVersion FROM Release_ WHERE servletContextName = 'com.liferay.journal.web';");
 
-		if (!(contains(${schemaVersion}, "0.0.1"))) {
+		if (!(contains(${schemaVersion}, "4.1.0"))) {
```
Comment (achaparro) [inline modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase:122]: @javalosjr96 from which module you got the 4.1.0 version? In this case we are queryin the journal web module, see JournalWebUpgradeStepRegistrator

File: modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/recorder/UpgradeRecorder.java [hunk 2]
```diff
@@ -95,9 +94,7 @@ public void recordErrorMessage(String loggerName, String message) {
 
 		messages.put(message, occurrences);
 
-		if (!_verifyProcessError &&
-			message.contains(VerifyException.class.getName())) {
-
+		if (!_verifyProcessError && loggerName.contains("verify")) {
```
Comment (achaparro) [inline modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/recorder/UpgradeRecorder.java:97]: @javalosjr96 checking just the verify string is weak, why is this needed?

**Comment by achaparro** [comment]
@javalosjr96 I have sent a new PR:
https://github.com/liferay-database-infra/liferay-portal/pull/2150

**Comment by achaparro** [comment]
ci:close
